src/test/java/org/opensearch/ad/transport/ADBatchAnomalyResultTransportActionTests.java

```diff
@@ -106,17 +106,14 @@ public void testHistoricalAnalysisWithValidDateRange() throws IOException, Inter
     }
 
     public void testHistoricalAnalysisWithNonExistingIndex() throws IOException {
-        ADBatchAnomalyResultRequest request = adBatchAnomalyResultRequest(
-            new DetectionDateRange(startTime, endTime),
-            randomAlphaOfLength(5)
-        );
+        ADBatchAnomalyResultRequest request = adBatchAnomalyResultRequest(new DateRange(startTime, endTime), randomAlphaOfLength(5));
```

Comment:
Spotless may complain down the road ...

Reply:
I ran spotless before sending a commit. Don't expect spotless will fail me :)

Reply:
:)